FFCDemoPaymentService/Data/ApplicationDbContext.cs

```diff
@@ -6,11 +6,20 @@ namespace FFCDemoPaymentService.Data
 {
     public class ApplicationDbContext : DbContext
     {
+        private readonly SchemaConfig schemaConfig;
         private readonly PostgresConnectionStringBuilder connectionStringbuilder;
 
+        public ApplicationDbContext() { }
         public ApplicationDbContext(PostgresConnectionStringBuilder stringBuilder) : base()
         {
             connectionStringbuilder = stringBuilder;
+            schemaConfig = null;
+        }
+
+        public ApplicationDbContext(PostgresConnectionStringBuilder stringBuilder, SchemaConfig schemaConfig) : base()
+        {
+            connectionStringbuilder = stringBuilder;
```

Comment:
bit of a nitpick, as was kind of an existing problem, but the 'b' in 'string builder' isn't consistently capitalised, sometimes it's `connectionStringbuilder` and other times it's `stringBuilder`

Reply:
:+1: I've made the change.